index.js

```diff
@@ -3,16 +3,33 @@ new Vue({
   data: {
   	todos: [
   		{ text: 'test', done: false, id: Date.now() }
-  	]
+  	], 
+    activeTodos: [], 
+    completedTodos: [],
+    showActive: [false],
+    showCompleted: [false]
   },
   methods: {
   	addTodo({target}){
     	this.todos.push({text: target.value, done: false, id: Date.now()})
       target.value = ''
+      console.log(this.todos)
     },
     removeTodo(id) {
-    	console.log('removeTodo has been clicked')
     	this.todos = this.todos.filter(todo => todo.id !== id)
-    }
+    },
+    updateActiveAndCompletedLists() {
+      this.completedTodos = this.todos.filter(todo => todo.done == true)
+      this.activeTodos = this.todos.filter(todo => todo.done == false)
+      this.todos = this.completedTodos
+      }
+      showActive() {
+      this.showActive = this.todos.filter(todo => todo.done == true)

```

Comment:
Above, I asked you to set `this.showActive` to  `true`. what is the filtering doing?